content/en/events/2022/kcsna/location.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Location & Venue"
+type: docs
+weight: 4
+---
+
+
+### Huntington Place
+
+Registration and badge pick-up will be available at the
+<a href="https://www.huntingtonplacedetroit.com/" rel="noopener noreferrer" target="_blank">Huntington Place Convention Center</a>.
+You **MUST** go through vaccination verification and pick up your badge before
```

Comment:
should we emphasize "before"?
```suggestion
You **MUST** go through vaccination verification and pick up your badge **BEFORE**
```

Reply:
removed the line about the vaccine req in general, the health and safety stuff is still TBD.